md/hardware-and-software-requirements.md

```diff
@@ -44,20 +44,22 @@ Software required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
 | Apple Safari | latest version |
 
 Notes:
-1. Your database must be configured to use the UTF-8 character set.
-2. MySQL must be configured with innoDB storage engine.
-3. There is a known issue between Bitronix (the transaction manager shipped by Bonitasoft for the Tomcat bundle and inside the Deploy bundle for Tomcat) and Microsoft SQL Server JDBC driver
+1. Bonita BPM can be executed on Java 8 and above. All development artifacts (connectors, REST API extensions, etc) must be compiled with Java 8 byte code (target version).
+2. Your database must be configured to use the UTF-8 character set.
+3. MySQL must be configured with innoDB storage engine.
+4. There is a known issue between Bitronix (the transaction manager shipped by Bonitasoft for the Tomcat bundle and inside the Deploy bundle for Tomcat) and Microsoft SQL Server JDBC driver
 (refer to: [MSDN note](https://msdn.microsoft.com/en-us/library/aa342335.aspx), [Bitronix note](http://bitronix-transaction-manager.10986.n7.nabble.com/Failed-to-recover-SQL-Server-Restart-td148.html)).
 Therefore, using Bitronix as a transaction manager with SQL Server does not work and is not supported. To use SQL Server database requires that you use the WildFly bundle provided by Bonitasoft.
-4. Bonita BPM can be executed on Java 8 and above. All development artifacts (connectors, REST API extensions, etc) must be compiled with Java 8 byte code (target version).
-5. Chrome version 60.0.3112 introduced an incompatibility impacting the functionality of the Bonita Portal. From that version onwards, once your BonitaBPM Platform is installed, please apply the following procedure to resolve this issue:
+5. Chrome version 60.0.3112 introduced an incompatibility impacting the functionality of Bonita BPM Portal. So from that Chrome version onwards, once Bonita BPM Platform is installed, apply the following procedure to resolve this issue:
+
+### Fixing procedure
 
 #### Subscription users
 ::: info
-**Note:** In order to apply this procedure, you MUST use a browser version that doesn't suffer from the incompatibility. You can use Firefox, Internet Explorer/Edge or Safari, for example.
+**Note:** In order to apply this procedure, you MUST use a Chrome browser version that doesn't suffer from the incompatibility, or you can use Firefox, Internet Explorer/Edge or Safari, for example.
 :::
 
-1. Log in the Bonita Portal as Administrator.
+1. Log in to Bonita BPM Portal as Administrator.
```

Comment:
Will need to remove all those `BPM` when merging to 7.6 => need a task, not to forget